tbx/settings/base.py

```diff
@@ -446,6 +446,10 @@
 # Override the Image class used by wagtailimages with a custom one
 WAGTAILIMAGES_IMAGE_MODEL = "images.CustomImage"
 
+pixel_limit = env.get("WAGTAILIMAGES_MAX_IMAGE_PIXELS")
+WAGTAILIMAGES_MAX_IMAGE_PIXELS = int(pixel_limit) if pixel_limit else 10_000_000
+
+
```

Comment:
@lparsons396 following on from our discussion in slack, I think we should amend this to use the image max upload size. If we update the settings so that value is actually read in and used, it should alert the editor if the image is too large - we can adjust the maximum size allowed if 2mb is still too much. I don't think that the number of pixels is a meaningful thing for editors to understand.

Reply:
Following on from this, it doesn't seem like any of the images used on a case study page that is crashing are particularly large. So either this issue is something else entirely, or the file size isn't the problem and these images do have a large pixel depth - so we need to find a way to identify the ones that are problematic but I am currently not sure how to check the image pixel depth in the shell.

Reply:
Further update: that case study *is* using a large image in the 'screenshots' and 'homepage image' field. We can discuss further at stand-up but another clean-up will be to remove those fields as they are not needed in the new site.

Reply:
It looks like you've quacked this, but I would have responded: "by particularly large, do you mean in pixels or bytes"? I've only been overhearing chat on this issue, but do we have any further traceback in the Sentry errors, other than memory quota vastly exceeded?

Reply:
We now have a separate plan in place to delete the images on the server that we know are causing crashes. In addition to Lauren's fix to set the maximum pixels (which can be a problem with animated gifs and we know that we have some animated gifs that are crashing the site), I have now enforced the WAGTAILIMAGES_MAX_UPLOAD_SIZE setting.